Description: Use system includes, remove -Werror=format-security
Author: Sergio Rubio <rubiojr@frameos.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- ruby-gtk-webkit-0.0.4.orig/ext/webkit/extconf.rb
+++ ruby-gtk-webkit-0.0.4/ext/webkit/extconf.rb
@@ -6,24 +6,11 @@ rescue LoadError
   use_gems = true
 end
 
-if use_gems or Object.const_defined?('Gem')
-  require 'rubygems'
-  gem 'glib2'
-  require 'mkmf-gnome2'
-  %w[rbglib.h rbgtk.h rbpango.h rbatk.h].each do |header|
-  	Gem.find_files(header).each do |f|
-		$CFLAGS += " '-I#{File.dirname(f)}'"
-	end
-  end
-end
-  %w[
- glib2 gdk_pixbuf2 atk gtk2].each do |package|
-  require package
-  $CFLAGS += " -I"+Gem.loaded_specs[package].full_gem_path+"/ext/"+package
-end
-if RbConfig::CONFIG.has_key?('rubyhdrdir')
-$CFLAGS += " -I" + RbConfig::CONFIG['rubyhdrdir']+'/ruby'
-end
+%w[rbglib.h rbgtk.h rbpango.h rbatk.h].each do |header|                       
+  $CFLAGS += " '-I#{RbConfig::CONFIG['vendorarchdir']}'" 
+end  
+$CFLAGS += " '-I#{RbConfig::CONFIG['rubyhdrdir']}/ruby'"
+$CFLAGS = $CFLAGS.gsub '-Werror=format-security', ''
 
 have_func("rb_errinfo")
 PKGConfig.have_package("gtk+-2.0") or exit(-1)
